tests/microos/networkmanager.pm

```diff
@@ -65,9 +67,16 @@ sub run {
         'mac_addr' => get_var('NICMAC'),
         'local_ip' => '10.0.2.15'
     );
-    %net_config = is_s390x ? %network_s390x : %network_qemu;
-    $nic_name = script_output("grep $net_config{'mac_addr'} /sys/class/net/*/address |cut -d / -f 5");
-
+    %net_config = is_svirt ? %network_s390x : %network_qemu;
+    if (is_s390x || is_qemu) {
+        $nic_name = script_output("grep $net_config{'mac_addr'} /sys/class/net/*/address |cut -d / -f 5");
+    } elsif (is_svirt_except_s390x) {
```

Comment:
This should directly target vmware only and not a broader group of cases.

Reply:
I agree with @mloviska on this point.